recipes/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "feedfinder2" %}
+{% set version = "0.0.4" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "3701ee01a6c85f8b865a049c30ba0b4608858c803fe8e30d1d289fdbe89d0efe" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+ 
+  run:
+    - python
+    
+
+about:
+  home: https://github.com/dfm/feedfinder2
+  license: MIT 
+  license_family: MIT
+  license_file: LICENSE
```

Comment:
The license file is not included in the source distribution. Please raise an issue upstream and put the link into the recipe as a comment (so that next time the feedstock is updated there is a visual reminder to check if the license has been subsequently included). 

You can then manually package the license as described here:

https://conda-forge.org/docs/meta.html#packaging-the-license-manually

Reply:
I have added this line:  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'

Reply:
I have added line:  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
Please let me know if there is any problem.